src/version.h.in

```diff
@@ -22,6 +22,13 @@
  * THE SOFTWARE.
  */
 // clang-format off
+#include <string>
+
 #define MIGRAPHX_VERSION_MAJOR @PROJECT_VERSION_MAJOR@
 #define MIGRAPHX_VERSION_MINOR @PROJECT_VERSION_MINOR@
+
+inline std::string get_migraphx_version() {
```

Comment:
Dont put this in the version header.

Reply:
Should i place it in `program.hpp` then?

Reply:
Yea